lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3170,19 +3169,32 @@ def model_add_material(model, material_name)
       end
     end
 
-    # OpenStudio::logFree(OpenStudio::Info, 'openstudio.standards.Model', "Adding material: #{material_name}")
-
     # Get the object data
-    data = model_find_object(standards_data['materials'], 'name' => material_name)
-    unless data
-      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.Model', "Cannot find data for material: #{material_name}, will not be created.")
-      return false
-      # @todo change to return empty optional material
+    if material_name.include?('Simple Glazing')
+      material_type = 'SimpleGlazing'
+      # default values
+      u_factor = 1.23
+      shgc = 0.61
+      vt = 0.81
+      material_name.split.each_with_index do |item, i|
+        if item == 'U'
+          u_factor = material_name[i + 1].to_f
+        elsif item == 'SHGC'
+          shgc = material_name[i + 1].to_f
+        elsif item == 'VT'
+          vt = material_name[i + 1].to_f
+        end
+      end
+    else
+      data = model_find_object(standards_data['materials'], 'name' => material_name)
```

Comment:
I think this is related to your first comment, I was not expecting that we would allow a straight look in the database, but I guess that seems reasonable.